src/logfile.c

```diff
@@ -53,7 +53,7 @@ static int logfile_config(const char *key, const char *value) {
     if (log_level < 0) {
       log_level = LOG_INFO;
       ERROR("logfile: invalid loglevel [%s] defaulting to 'info'", value);
-      return 1;
+      return 0;
```

Comment:
That was 'error' by code design. Developer explicitly choosed to return "1".
Now you want to return "0", that does message not so critical as it was before.
I think message log level should be changed to .... WARNING or NOTICE ("project" should have policy about such case).

Reply:
I agree with warning